linode/ssh.go

```diff
@@ -12,6 +12,7 @@ func commHost(state multistep.StateBag) (string, error) {
 func sshConfig(state multistep.StateBag) (*ssh.ClientConfig, error) {
 	return &ssh.ClientConfig{
 		User: "root",
+		HostKeyCallback: ssh.InsecureIgnoreHostKey(),
```

Comment:
Everything looks good, except I think this line shouldn't be the default. It can certainly be made configurable, though.

Apologies for the delay on this.

Reply:
ok true, will work on generating the hostkey if passed in.

Reply:
Being able to specify an SSH key to verify against is a good idea, but I also think the insecure option should be strictly opt-in, e.g. `"ssh_insecure": true`. If neither is provided, it would use the current behavior, which is to not specify a value for `HostKeyCallback` at all.

Reply:
not specifying HostKeyCallback will automatically fail on latest Go version since the behaviour has been changed, according to https://github.com/golang/go/issues/19767 

Insecure is the same as not specifying AFAIK. This should be the default, as was previously, but I understand ensuring that a user explicitly states that they want to use the insecure mode. 

Is it ok for the connection attempt to fail if "ssh_insecure":true is not set?